src/Propel/Common/Config/PropelConfiguration.php

```diff
@@ -124,6 +124,8 @@ protected function addDatabaseSection(ArrayNodeDefinition $node)
                                     ->end()
                                     ->arrayNode('attributes')
                                         ->children()
+                                            ->booleanNode('PROPEL_ATTR_CACHE_PREPARES')->defaultFalse()->end()
+                                            ->booleanNode('PropelPDO::PROPEL_ATTR_CACHE_PREPARES')->defaultFalse()->end()
```

Comment:
According to the documentation you can set it either with `PropelPDO::` or without.